sycl/doc/extensions/C-CXX-StandardLibrary/C-CXX-StandardLibrary.rst

```diff
@@ -270,6 +280,6 @@ Every set of functions is implemented in a separate fallback
 library. For example, a fallback for `cl_intel_devicelib_cassert`
 extension is provided as `libsycl-fallback-cassert.spv`
 
-NOTE that AOT compilation is not yet supported. Driver will have to
-check for extension support and link the corresponding SPIR-V fallback
-implementation, but this is not implemented yet.
+For AOT compilation, fallback libraries are also provided as object
+files (e.g. `libsycl-fallback-cassert.o`). Device code in these object
+files is identical to the the `*.spv` files.
```

Comment:
I'd expect the device code in .o files to be llvm IR bitcode, if it is produced with '-fsycl -c'.
Is this really a SPIR-V now?

Reply:
@vzakhari is correct. I will reword this section.

Reply:
Done. Please check if it is OK now.

Reply:
LGTM, thanks